src/examples/browser.yml

```diff
@@ -1,13 +1,17 @@
 description: >
-  Runs Cypress tests in specific browser.
+  Runs Cypress tests in specific (non-edge) browser.
```

Comment:
Maybe something like this so we start to connect the dots regarding Edge support? 

Run Cypress tests using specified browser. The (circleci/browser-tools) [https://circleci.com/developer/orbs/orb/circleci/browser-tools#commands-install-browser-tools] orb currently supports Chrome & Firefox. See [link to Install command], for Microsoft Edge support.